caas/kubernetes/provider/k8s.go

```diff
@@ -500,10 +500,14 @@ please choose a different hosted model name then try again.`, hostedModelName),
 		)
 	}
 
+	podArch := arch.AMD64
+	if args.BootstrapConstraints.HasArch() {
+		podArch = *args.BootstrapConstraints.Arch
+	}
 	return &environs.BootstrapResult{
-		// TODO(bootstrap): review this default arch and series(required for determining DataDir etc.) later.
-		Arch:                   arch.AMD64,
-		Series:                 "kubernetes",
+		Arch: podArch,
+		// TODO(wallyworld) - use actual series of controller pod image
+		Series:                 series.LatestLts(),
```

Comment:
Does series.LatestLts() return `kubernetes` or a real series?

Reply:
As discussed, we don't use this "kubernetes" series at all, so it's ok to set it to a real series.